spec.md

```diff
@@ -540,11 +540,28 @@ The `requires` field in the [Catalog](#catalog-mgmt) endpoint enables a platform
 
 #### Route Services
 
-There are a class of service offerings that intermediate requests to applications, performing functions such as rate limiting or authorization. To configure a service instance with behavior specific to an application's routable address, a broker client MAY send the address along with the request to create a binding using `"bind_resource":{"route":"some-address.com"}`.
+There are a class of service offerings that intermediate requests to
+applications, performing functions such as rate limiting or authorization.
+If the platform and service both support this feature then the following
+applies:
 
-Some platforms MAY support proxying of application requests to service instances. In this case the platform needs to know where to send application requests; to facilitate this, the broker MUST return a `route_service_url` in the response for a request to create a binding. Not all services of this type expect to receive requests proxied by the platform; some services will have been configured out-of-band to intermediate requests to applications. In this case, the broker will not return `route_service_url` in response to the create binding request. By sending `bind-resource` as described above, the platform enables dynamic configuration of a service instance already in the application request path for the route, requiring no change in the platform routing tier.
+- To configure a service instance with behavior specific to an application's
+  routable address, a platform MUST send the address along with the request
+  to create a binding using `"bind_resource":{"route":"some-address.com"}`.
 
-The `requires` field in the [Catalog](#catalog-mgmt) endpoint enables a platform marketplace to validate requests to create bindings. A platform MAY reject requests to create bindings when a broker has declared `"requires":["route_forwarding"]` for a service in the catalog endpoint.
+- Upon receiving an incoming appliation request, the platform needs to know
+  where to send this request to. To facilitate this, the broker MUST return
+  a `route_service_url` in the response for a request to create a binding
+  unless there has been some out-of-band configuration already performed;
+  in which case the broker MUST NOT return a `route_service_url` parameter.
+
```

Comment:
I like the idea of splitting the sentence into two.  However, when I moved the 2nd sentence to a new bullet it felt a little odd because those two sentences really need to go hand-in-hand to make sure people realize the MUST has a conditional aspect to it.  see what you think of my edits and let me know if you still think a 2nd bullet instead of a second sentence is still needed.